shared/storage/minio.py

```diff
@@ -80,6 +85,25 @@ def init_minio_client(
         if port is not None:
             host = "{}:{}".format(host, port)
 
+        timeout = timedelta(minutes=5).seconds
+        http_client = urllib3.PoolManager(
+            timeout=Timeout(connect=timeout, read=timeout),
```

Comment:
5 minutes for `connect` is unreasonable. even for `read`, this might be a bit too generous.
for native symbolicator, we assumed a "low end" read throughput of ~4M/s, meaning that it fetches 1G within ~4 minutes.
as we are talking about *internal* storage here, we can set the expectations towards throughput a lot higher, and thus the timeouts a bit lower.